app/code/community/Bolt/Boltpay/Model/Cron.php

```diff
@@ -24,30 +24,42 @@ class Bolt_Boltpay_Model_Cron
 {
     use Bolt_Boltpay_BoltGlobalTrait;
 
-    /*
+    /**
      * We will have a conservative tolerance of 30 minutes for non-confirmed pre-auth orders before we allow the system
      * to remove them
      */
     const PRE_AUTH_STATE_TIME_LIMIT_MINUTES = 30;
 
+    /**
+     * 60*60*24*7*2=1209600 is the amount of time in seconds we allow immutable quotes to exist before we delete them
```

Comment:
Comment is clear but an even clearer one would include "2 weeks" somewhere I think.